Sources/EduIDExpansion/Flows/Onboarding/OnboardingCoordinator.swift

```diff
@@ -0,0 +1,59 @@
+import UIKit
+
+final class OnboardingCoordinator: OnboardingCoordinatorType, OnBoardingNavigationDelegate, ScanMainNavigationDelegate {
+    
+    weak var navigationController: UINavigationController!
+    weak var delegate: OnBoardingMainNavigationDelegate?
+    
+    var children: [CoordinatorType] = []
+    
+    var currentScreenType: ScreenType = .landingScreen
+    
+    func start(presentOn viewController: UIViewController) {
+        
+        let landingScreen = OnBoardingLandingPageViewController()
+        landingScreen.screenType = .landingScreen
+        landingScreen.delegate = self
+        
+        let navigationController = UINavigationController(rootViewController: landingScreen)
+        navigationController.modalTransitionStyle = .flipHorizontal
+        
+        self.navigationController = navigationController
+        
+//        viewController.present(self.navigationController, animated: false)
```

Comment:
Commented Out Code Without Comment (cocwoc)

Reply:
I changed this

Reply:
I don't see a change? Code is still commented out.